bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx88-en.html

```diff
@@ -0,0 +1,108 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% extends "firefox/whatsnew/base.html" %}
+
+{% block page_title %}What’s new with Firefox{% endblock %}
+
+{#- This will appear as <meta property="og:description"> which can be used for social share -#}
+{% block page_og_desc %}What’s new with Firefox{% endblock %}
+
+{% block body_id %}firefox-whatsnew{% endblock %}
+{% block body_class %}{% endblock %}
+
+{% block page_css %}
+  {{ css_bundle('firefox_whatsnew_88_en') }}
+{% endblock %}
+
+{% block site_header %}{% endblock %}
+
+{% block content %}
+<main class="content-wrapper mzp-t-firefox">
+  <header class="c-page-header">
+    <div class="mzp-l-content c-page-header-inner">
+      <h2 class="c-page-header-logo-fx">{{ ftl('whatsnew-firefox-browser') }}</h2>
+      <div class="mzp-c-notification-bar mzp-t-success up-to-date">
+        <p>Congrats! You’re using now version <span data-audio="/media/img/firefox/whatsnew/whatsnew88-en/jingle.mp3" class="flux" id="outatime">88</span> of Firefox.</p>
+      </div>
+      <div class="mzp-c-notification-bar out-of-date">
+        <p>{{ ftl('whatsnew-out-of-date-notification') }}</p>
+      </div>
+    </div>
+  </header>
+
+  {% set landing_source = '?source=' + campaign if campaign else '' %}
+
+  <section class="wnp-content-main">
+    <div class="mzp-l-content mzp-t-content-lg">
+      {% if variant == '2' %}
+        <img class="wnp-main-image" src="{{ static('img/firefox/whatsnew/whatsnew88-en/hero.png') }}" width="620" alt="">
+      {% elif variant == '3' %}
+        <video poster="{{ static('img/firefox/whatsnew/whatsnew88-en/hero.png') }}" class="c-animated-graphic wnp-vid-1" muted autoplay loop>
+          <source type="video/webm" src="https://www-dev.allizom.org/media/img/firefox/whatsnew/whatsnew88/video.794560d1ad14ae53fd3b56e2fc8dc70f.webm" media="screen and (prefers-color-scheme: light)">
```

Comment:
No need for the `media` attribute now, since it didn't work anyway.

Reply:
I assume we need to update the video URLs for production?

Reply:
should be fine to use relative urls e.g. `/media/img/firefox/whatsnew/whatsnew88/video.794560d1ad14ae53fd3b56e2fc8dc70f.webm`

Reply:
Revised.